### ISSUE TITLE
[Bug]: Validation error with broker_client.get_account_by_id

### ISSUE THREAD

**Issue by xuru** [alpacahq/alpaca-py#163]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Pydantic is failing validation of `Disclosures` when getting an account by ID.  It works when fetching all accounts however.

```
broker_client = BrokerClient(api_key=settings.APCA_API_KEY_ID, secret_key=settings.APCA_API_SECRET_KEY, sandbox=True)
broker_client.list_accounts(search_parameters=ListAccountsRequest(**{'query': email}))
Out[29]: 
[{   'account_number': '953026141',
     'agreements': None,
     'contact': None,
     'created_at': '2021-11-11T14:40:04.290975Z',
     'crypto_status': <AccountStatus.INACTIVE: 'INACTIVE'>,
     'currency': 'USD',
     'disclosures': None,
     'documents': None,
     'id': UUID('512d823b-c7d7-43e3-8bb1-68a196033272'),
     'identity': None,
     'last_equity': '-1561.3',
     'status': <AccountStatus.ACTIVE: 'ACTIVE'>,
     'trusted_contact': None}]
```
Also, note that disclosures is None, when there clearly is one.
```
broker_client.get_account_by_id(account_id='512d823b-c7d7-43e3-8bb1-68a196033272')
Traceback (most recent call last):
  File "/Users/eric/.virtualenvs/wedge-backend-ghk8MMIr-py3.10/lib/python3.10/site-packages/IPython/core/interactiveshell.py", line 3398, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-30-5b477e44e7b8>", line 1, in <cell line: 1>
    broker_client.get_account_by_id(account_id='512d823b-c7d7-43e3-8bb1-68a196033272')
  File "/Users/eric/.virtualenvs/wedge-backend-ghk8MMIr-py3.10/lib/python3.10/site-packages/alpaca/broker/client.py", line 191, in get_account_by_id
    return Account(**resp)
  File "/Users/eric/.virtualenvs/wedge-backend-ghk8MMIr-py3.10/lib/python3.10/site-packages/alpaca/broker/models/accounts.py", line 269, in __init__
    parse_obj_as(Disclosures, response["disclosures"])
  File "pydantic/tools.py", line 38, in pydantic.tools.parse_obj_as
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for ParsingModel[Disclosures]
__root__ -> employment_status
  value is not a valid enumeration member; permitted: 'unemployed', 'employed', 'student', 'retired' (type=type_error.enum; enum_values=[<EmploymentStatus.UNEMPLOYED: 'unemployed'>, <EmploymentStatus.EMPLOYED: 'employed'>, <EmploymentStatus.STUDENT: 'student'>, <EmploymentStatus.RETIRED: 'retired'>])

```
Firing up the debugger, in the constructor for the Account model, the raw data for the disclosure is:
```
{'employment_status': 'UNEMPLOYED', 'employer_address': 'Z', 'employment_position': 'Y', 'is_control_person': False, 'is_affiliated_exchange_or_finra': False, 'is_politically_exposed': False, 'immediate_family_exposed': False, 'is_discretionary': False}
```
Manually validating with pydantic gives the same error:
```
Disclosures(**{'employment_status': 'UNEMPLOYED', 'employer_address': 'Z', 'employment_position': 'Y', 'is_control_person': False, 'is_affiliated_exchange_or_finra': False, 'is_politically_exposed': False, 'immediate_family_exposed': False, 'is_discretionary': False})
Traceback (most recent call last):
  File "/Users/eric/.virtualenvs/wedge-backend-ghk8MMIr-py3.10/lib/python3.10/site-packages/IPython/core/interactiveshell.py", line 3398, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-33-6b7ec583def4>", line 1, in <cell line: 1>
    Disclosures(**{'employment_status': 'UNEMPLOYED', 'employer_address': 'Z', 'employment_position': 'Y', 'is_control_person': False, 'is_affiliated_exchange_or_finra': False, 'is_politically_exposed': False, 'immediate_family_exposed': False, 'is_discretionary': False})
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for Disclosures
employment_status
  value is not a valid enumeration member; permitted: 'unemployed', 'employed', 'student', 'retired' (type=type_error.enum; enum_values=[<EmploymentStatus.UNEMPLOYED: 'unemployed'>, <EmploymentStatus.EMPLOYED: 'employed'>, <EmploymentStatus.STUDENT: 'student'>, <EmploymentStatus.RETIRED: 'retired'>])

```

### Expected Behavior

When fetching an account by id, it should validate correctly and construct the Account model.

### SDK Version I encountered this issue in

alpaca-py version: 0.6.0

### Steps To Reproduce

```markdown
Add an account to sandbox or use an existing one that has a disclosure, then:
1. broker_client = BrokerClient(api_key=settings.APCA_API_KEY_ID, secret_key=settings.APCA_API_SECRET_KEY, sandbox=True)
2. broker_client.get_account_by_id(account_id='<id of account>')
```

### Filled out the Steps to Reproduce section?

- [X] I have entered valid steps to reproduce my issue or have attached a minimally reproducible case in code that shows my issue happening; and understand that without this my issue will be flagged as invalid and closed after 30 days.

**Comment by forLinDre** [comment]
I get the same error using TradingClient's close_all_positions method.

**Comment by haxdds** [comment]
Fixed by #235
